docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -3,189 +3,101 @@ title: Microsoft Teams
-- `chat_messages`: messages sent to the signed in user's chats.
-- `channels`: channels that the signed in user is a member of.
-- `channel_messages`: messages sent to the signed in user's channels.
-
-## Chatbot
-
-While the Microsoft Teams handler allows you to read/send messages to a chat or channel as shown above, it is also possible to create a chat bot that will listen to messages sent to a chat or channel and respond to them.
-
-### Step 1: Create a Microsoft Teams Data Source
+Required connection parameters include the following:
 
-As shown above, create a database with the new `teams` engine by passing in the required parameters:
-
-```sql
-CREATE DATABASE teams_datasource
-WITH ENGINE = 'teams',
-PARAMETERS = {
-  "client_id": "your-client-id",
-  "client_secret": "your-client-secret",
-  "tenant_id": "your-tenant-id"
-};
-```
+* `client_id`: The client ID of the registered Microsoft Entra ID application.
+* `client_secret`: The client secret of the registered Microsoft Entra ID application.
 
 <Note>
-The chat bot will assume the identity of the user who signed in with their Microsoft account when the database was created.
+Microsoft Entra ID was previously known as Azure Active Directory (Azure AD).
 </Note>
 
-### Step 2: Create an [Agent](https://docs.mindsdb.com/agents/agent)
-
-An agent is created by combining a conversational model, like [LlamaIndex](https://docs.mindsdb.com/integrations/ai-engines/llamaindex), with a set of Skills.
-
-Here, we will create an agent without any skills, however, it is possilbe to create agents with skills such as the ability to answer questions from a knowledge base.
-
-#### Step 2.1: Create a Conversational Model
-
-Create a conversational model using either the [`LangChain`](https://docs.mindsdb.com/integrations/ai-engines/langchain) or [`LlamaIndex`](https://docs.mindsdb.com/integrations/ai-engines/llamaindex) integrations. Given below is an example of a conversational model created using the `LlamaIndex` integration:
-
-```sql
-CREATE ML_ENGINE llama_index_engine
-FROM llama_index
-USING openai_api_key='your-openai-api-key';
-```
-
-```sql
-CREATE MODEL llama_index_convo_model
-PREDICT answer
-USING
-  engine = 'llama_index_engine',
-  mode = 'conversational',
-  prompt = 'answer users questions as a helpful assistant',
-  user_column = 'question',
-  assistant_column = 'answer';
-```
-
-#### Step 2.2: Create an Agent using the Conversational Model
-
-Let's create an agent using the conversational model created above:
-
-```sql
-CREATE AGENT convo_agent
-USING
-    model = 'llama_index_convo_model',
-```
-
-### Step 3: Create a [Chatbot](https://docs.mindsdb.com/agents/chatbot)
-
-Finally, create a chatbot using the agent and the Microsoft Teams data source created above:
-
-```sql
-CREATE CHATBOT teams_chatbot
-USING
-    database = 'teams_datasource',
-    agent = 'convo_agent';
-```
+Follow the instructions given below to set up the Microsoft Teams app that will act as the chatbot:
```

Comment:
Let's enclose lines 38-80 in this accordion like in Slack docs.
```
<Accordion title="Set up a Microsoft Teams app as a chatbot">
lines 38-80
</Accordion>
```

Reply:
Should we still do this given my comment above above about the lack of tables (and a Usage section)?

Reply:
@martyna-mindsdb I've added this note and encapsulated the instructions for setting up the chat-bot in a Usage section. Let me know what you think.